edx_proctoring/locale/config.yaml

```diff
@@ -71,14 +71,3 @@ locales:
 # The locales used for fake-accented English, for testing.
 dummy_locales:
     - eo
-
-# Directories we don't search for strings.
-ignore_dirs:
-    - docs
-    - edx-proctoring
-    - logs
-    - node_modules
-    - performance
-    - requirements
-    - scripts
-    - settings
```

Comment:
Good question. It is because we are now cd'ing to the Django app directory and running the scripts there. These directories are all in the root, so they aren't seen now.